module/solutions/helper.js

```diff
@@ -2365,7 +2365,7 @@ module.exports = class SolutionsHelper {
 				let searchQuery = []
 
 				if (search !== '') {
-					searchQuery = [{ name: new RegExp(search, 'i') }, { description: new RegExp(search, 'i') }]
+					searchQuery = [{ title: new RegExp(search, 'i') }, { description: new RegExp(search, 'i') }]
```

Comment:
@Prajwal17Tunerlabs why name is changed to title

Reply:
in projects model, we are using `title` instead of `name`